<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_load_citra">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Citra</string>
    </property>
   </widget>
   <widget class="QLabel" name="load_citra">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>411</width>
      <height>291</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hasil_citra">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>421</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>440</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Hasil</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1067</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperasi_Titik">
    <property name="title">
     <string>Operasi Titik</string>
    </property>
    <addaction name="actionGrayscale"/>
    <addaction name="actionBrightness"/>
    <addaction name="actionSimple_Contrast"/>
    <addaction name="actionContrast_Stretching"/>
    <addaction name="actionNegative"/>
    <addaction name="actionBiner"/>
   </widget>
   <widget class="QMenu" name="menuHistogram">
    <property name="title">
     <string>Histogram</string>
    </property>
    <addaction name="actionHistogram_Grayscale"/>
    <addaction name="actionHistogram_RGB"/>
    <addaction name="actionHistogram_Equalization"/>
   </widget>
   <widget class="QMenu" name="menuGeometri">
    <property name="title">
     <string>Geometri</string>
    </property>
    <addaction name="actionTranslasi"/>
    <addaction name="actionRotasi"/>
    <addaction name="actionResize"/>
    <addaction name="actionCrop"/>
   </widget>
   <addaction name="menuOperasi_Titik"/>
   <addaction name="menuHistogram"/>
   <addaction name="menuGeometri"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionBrightness">
   <property name="text">
    <string>Brightness</string>
   </property>
  </action>
  <action name="actionSimple_Contrast">
   <property name="text">
    <string>Simple Contrast</string>
   </property>
  </action>
  <action name="actionContrast_Stretching">
   <property name="text">
    <string>Contrast Stretching</string>
   </property>
  </action>
  <action name="actionNegative">
   <property name="text">
    <string>Negative</string>
   </property>
  </action>
  <action name="actionBiner">
   <property name="text">
    <string>Biner</string>
   </property>
  </action>
  <action name="actionHistogram_Grayscale">
   <property name="text">
    <string>Histogram Grayscale</string>
   </property>
  </action>
  <action name="actionHistogram_RGB">
   <property name="text">
    <string>Histogram RGB</string>
   </property>
  </action>
  <action name="actionHistogram_Equalization">
   <property name="text">
    <string>Histogram Equalization</string>
   </property>
  </action>
  <action name="actionRotasi">
   <property name="text">
    <string>Rotasi</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="text">
    <string>Resize</string>
   </property>
  </action>
  <action name="actionTranslasi">
   <property name="text">
    <string>Translasi</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
